en/sql/query/truncate.rst

```diff
@@ -8,19 +8,25 @@ TRUNCATE
 
 You can delete all records in the specified table by using the **TRUNCATE** statement.
 
-This statement internally delete first all indexes and constraints defined in a table and then deletes all records. Therefore, it performs the job faster than using the **DELETE FROM** *table_name* statement without a **WHERE** clause.
+It has following advantages than using the **DELETE FROM** *table_name* statement without a **WHERE** clause.
 
-If the **PRIMARY KEY** constraint is defined in the table and this is referred by one or more **FOREIGN KEY**, it follows the **FOREIGN KEY ACTION**. If the **ON DELETE** action of **FOREIGN KEY** is **RESTRICT** or **NO_ACTION**, the **TRUNCATE** statement returns an error. If it is **CASCADE**, it deletes **FOREIGN KEY**. The **TRUNCATE** statement initializes the **AUTO INCREMENT** column of the table. Therefore, if data is inserted, the **AUTO INCREMENT** column value increases from the initial value.
+* It's way faster due to deleting all indexes and constraints in adavance and deleing records at once.
+* There is no vacuum cost.
+* It generates way lesser log records so that it is better in terms of HA, Recovery and Rollback.
 
 .. note:: 
 
-    To execute the **TRUNCATE** statement, the authorization of **ALTER**, **INDEX**, and **DELETE** is required on the table. For granting authorization, see :ref:`granting-authorization`.
+    * The **ON DELETE** trigger is disabled while deleting records with the **TRUNCATE** statement.
```

Comment:
Does this sentence mean that the trigger does not work when the truncate statement is executed? 
Or does it mean that the ON DELETE action does not work if only the truncate statement is executed without CASCADE ?

Reply:
If what you mean is a deleting trigger, you will modify it as the following.
* The deleting trigger is disabled while deleting records with the **TRUNCATE** statement.

Otherwise, it would be better to modify the trigger to an action as the following.
* The **ON DELETE** action is disabled while deleting records with the **TRUNCATE** statement.

Reply:
It means trigger. I'll follow your instruction. Thank you.

Reply:
How about **DELETE** trigger? which seems to be what the manual call it.

Reply:
@mhoh3963 How about **DELETE** trigger? which seems to be what the manual calls it.

Reply:
OK, it is good.